ansible/roles/pulp_site/defaults/main.yml

```diff
@@ -0,0 +1,43 @@
+pulp_site_url: "{{ appliances_pulp_url }}"
+pulp_site_port: 8080
+pulp_site_username: admin # shouldn't be changed
+pulp_site_upstream_content_url: https://ark.stackhpc.com/pulp/content
+pulp_site_upstream_username: slurm-app-ci
+pulp_site_upstream_password: "{{ lookup('ansible.builtin.env', 'ARK_PASSWORD') }}"
```

Comment:
Nope, I don't think we should look this up from env. It should normally be set as vaulted group_vars, so should be empty here.

Maybe we could help signpost this by adding:

```yaml
# environments/common/inventory/group_vars/pulp_site:
# Add your creds for stackhpc ark here
# pulp_site_upstream_username:
# pulp_site_upstream_password: 
```

Reply:
this is to get the gh secret from CI but yeah will move to .stackhpc